lectures/time_series_with_matrices.md

```diff
@@ -49,7 +49,9 @@ We will use the following imports:
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib import cm
-plt.rcParams["figure.figsize"] = (11, 5)  #set default figure size
+plt.rcParams["figure.figsize"] = (11, 5)  # Set default figure size
```

Comment:
@HumphreyYang can we remove this?

Reply:
Hi @mmcky,

I just experimented with deleting it, but it squeezed the time-series plot too much.

I think we might wan to keep it.

Reply:
roger that @HumphreyYang can we then please make the size specific to the figure in question (as a special case).

Reply:
I think most of the graphs below are time series as it is the theme of the lecture, but I will check if it can be done.

Reply:
Hi @mmcky,

I think the graphs below are all time-series plots that I think would be best presented in the current ratio. Perhaps this is why it is not removed in our previous iteration : )

Reply:
@HumphreyYang roger that. I wonder if there is some way to indicate that in the comment so we don't keep having this conversation :-)

Reply:
Perhaps we can `# Set default figure size #` above the line so we know this is final decision : )

Reply:
Thanks @HumphreyYang lets change the comment to `# custom figsize for this lecture`